frontend/src/metabase-lib/expressions/suggest.ts

```diff
@@ -82,7 +82,12 @@ export function suggest({
       if (name && database) {
         const helpText = getHelpText(name, database, reportTimezone);
         if (helpText) {
-          return { suggestions, helpText };
+          const clause = MBQL_CLAUSES[helpText?.name];
+          const isSupported =
+            !clause || database?.hasFeature(clause.requiresFeature);
```

Comment:
Shouldn't this be the other way around?

```suggestion
            clause && database?.hasFeature(clause.requiresFeature);
```

Reply:
I'm being conservative here: we still want to return help text even when the clause is not defined in `MBQL_CLAUSES` but _is_ defined in [`HELPER_TEXT_STRINGS`](https://github.com/metabase/metabase/blob/master/frontend/src/metabase-lib/expressions/helper-text-strings.ts#L39).

So if there is no clause found (ie. `!clause`) we allow the help text to be returned.